axiom/nr_txn.c

```diff
@@ -3457,3 +3459,24 @@ void nr_txn_record_log_event(nrtxn_t* txn,
 
   nr_txn_add_logging_metrics(txn, log_level_name);
 }
+
+extern void nr_txn_add_php_package(nrtxn_t* txn,
```

Comment:
No need for `extern` here.

Reply:
Fixed: [9325d71](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/9325d719fcc1c0fae33a1f73ca9bc6eae6d74689)